test/test_hid_transport.py

```diff
@@ -0,0 +1,85 @@
+import os
+import unittest
+from u2flib_host import hid_transport
+from u2flib_host import exc
+from u2flib_host.yubicommon.compat import byte2int, int2byte
+from mock import patch
+
+class TestHIDDevice():
```

Comment:
Please use new-style classes for consistency.

Reply:
Not a python guru, have been using it on and off since 2000. So, i kinda rely on linters for style guidance. Do you recommend any linters?